org.maven.ide.eclipse.ajdt/src/org/maven/ide/eclipse/ajdt/AjdtProjectConfigurator.java

```diff
@@ -199,16 +202,18 @@ public void configureRawClasspath(ProjectConfigurationRequest request, IClasspat
     }
   }
 
-  @Override
-  protected File[] getSourceFolders(ProjectConfigurationRequest request, MojoExecution mojoExecution)
+  // TODO: Where was this formerly protected method used in the old parent class? What is being used instead now?
```

Comment:
@joker1, can you look at this section and maybe answer the question?

Reply:
I took a quick look at this.

It seems that AbstractJavaProjectConfigurator is no longer available. 

AjdtProjectConfigurator needs to extend from AbstractProjectConfigurator instead. May look this further when I have enough time.